src/styles/main-window-glas.css

```diff
@@ -0,0 +1,91 @@
+#app {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+
+  background: linear-gradient(to bottom, #603e9f, #2e0c4b); /*후보1*/
+  background: linear-gradient(to right, #a5a4a4, #3b3c3c);
+  background: linear-gradient(to bottom, #bdc3c7, #2c3e50);
+  background: linear-gradient(to right, #e6dada, #274046);
+  background: linear-gradient(to right, #283048, #859398);
+}
```

Comment:
S. Review in balance-window-glas.css

Reply:
Wird bis zum nächsten Pull Request erledigt.